vmdk_plugin/E2E_Tests/e2e_verifyVolmProperties_test.go

```diff
@@ -0,0 +1,177 @@
+// Copyright 2017 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This is an end-to-end test. Test will ssh to a vm and create a volume using docker cli.
+// After creating a volume, test will ssh to the ESX and verify the properties of the volume.
+// Properties being verified - capacity, disk-format and vm-attached field.
```

Comment:
Good comment ! 
nit: Our test assume SSH certs are set up for password-less login - maybe worth mentioning it here.